src/gsy_e/models/strategy/energy_parameters/energy_params_eb.py

```diff
@@ -0,0 +1,182 @@
+from logging import getLogger
+from typing import Dict, Optional
+
+import pendulum
+from gsy_framework.utils import convert_kW_to_kWh
+
+from gsy_e.gsy_e_core.enums import AvailableMarketTypes
+from gsy_e.models.state import LoadState
+from gsy_e.models.strategy.energy_parameters.standard_profile_builders import StandardProfileParser
+
+logger = getLogger(__name__)
+
+
+class StandardProfileException(Exception):
+    """Exception for the StandardProfile."""
+
+
+class LoadSSPEnergyParameters:
+    """Energy parameters of the load strategy for the forward markets."""
+
+    _STANDARD_SOLAR_PROFILE = StandardProfileParser.parse()
+
+    def __init__(self, capacity_kW):
+        self.state = LoadState()
+        self.capacity_kW: float = capacity_kW
+
+        self._area = None
+        self._profile_generator: Optional[ForwardTradeProfileGenerator] = None
+
+    def serialize(self):
+        """Return dict with the current energy parameter values."""
+        return {
+            "capacity_kW": self.capacity_kW
+        }
+
+    def event_activate_energy(self, area):
+        """Update energy requirement upon the activation event."""
+        self._area = area
+        self._profile_generator = ForwardTradeProfileGenerator(
+            peak_kWh=convert_kW_to_kWh(self.capacity_kW, self._area.config.slot_length))
+
+    def event_traded_energy(self, energy_kWh, market_slot: pendulum.DateTime, product_type):
```

Comment:
This method will change based on the actual actions that we want to take on trades. I've just used this name as a way to show how the profile generator would be called with traded energy values.